docs/en-us/1.3.5/user_doc/expansion-reduction.md

```diff
@@ -0,0 +1,256 @@
+There are two steps for shrinking. After performing the following two steps, the shrinking operation can be completed.
+
+### 2.1 Stop the service on the scaled-down node
+ * If you are scaling down the master node, identify the physical machine where the master service is located, and stop the master service on the physical machine.
+ * If the worker node is scaled down, determine the physical machine where the worker service is to be scaled down and stop the worker and logger services on the physical machine.
+ 
+```shell
+# stop command:
+bin/stop-all.sh # stop all services
+
+sh bin/dolphinscheduler-daemon.sh stop master-server  # stop master service
+sh bin/dolphinscheduler-daemon.sh stop worker-server  # stop worker service
+sh bin/dolphinscheduler-daemon.sh stop logger-server  # stop logger service
+sh bin/dolphinscheduler-daemon.sh stop api-server     # stop api    service
+sh bin/dolphinscheduler-daemon.sh stop alert-server   # stop alert  service
+
+
+# start command:
+bin/start-all.sh # start all services
+
+sh bin/dolphinscheduler-daemon.sh start master-server # start master service
+sh bin/dolphinscheduler-daemon.sh start worker-server # start worker service
+sh bin/dolphinscheduler-daemon.sh start logger-server # start logger service
+sh bin/dolphinscheduler-daemon.sh start api-server    # start api    service
+sh bin/dolphinscheduler-daemon.sh start alert-server  # start alert  service
+
+```
+
+```
+ Attention: When using stop-all.sh or stop-all.sh, if the machine without the command is not configured to be ssh-free for all machines, it will prompt for the password.
+```
+
+- After the script is completed, use the `jps` command to see if each node service was successfully shut down (`jps` comes with the `Java JDK`)
+
+```
+    MasterServer         ----- master service
+    WorkerServer         ----- worker service
+    LoggerServer         ----- logger service
+    ApiApplicationServer ----- api    service
+    AlertServer          ----- alert  service
+```
+If the corresponding master service or worker service does not exist, then the master/worker service is successfully shut down.
+
+
+### 2.2 Modify the configuration file
+
+ - modify the configuration file `conf/config/install_config.conf` on the **all** nodes, synchronizing the following configuration.
+    
+    * to scale down the master node, modify the ips and masters parameters.
+    * to scale down worker nodes, modify the ips and workers parameters.
+
+```shell
+# which machines to deploy DS services on, "localhost" for this machine
+ips="ds1,ds2,ds3,ds4"
+
+# ssh port,default: 22
+sshPort="22"
+
+# which machine the master service is deployed on
+masters="existing master01,existing master02,ds1,ds2"
+
+# The worker service is deployed on which machine, and specify which worker group this worker belongs to, the following example of "default" is the group name
+workers="existing worker01:default,existing worker02:default,ds3:default,ds4:default"
+
+```
+
+
```

Comment:
Extra blank lines can be deleted

Reply:
Thanks, the extra blank lines have been deleted, please check